src/Microsoft.DotNet.Migration/project.json

```diff
@@ -0,0 +1,72 @@
+{
+  "version": "1.0.0-featmsbuild-*",
+  "buildOptions": {
+    "warningsAsErrors": true
+  },
+  "dependencies": {
+    "dotnet": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.ProjectModel": {
+      "target": "project"
+    },
+    "NuGet.Commands": {
+      "version": "3.5.0-rc1-1697",
+      "exclude": "compile"
+    },
+    "NuGet.CommandLine.XPlat": "3.5.0-rc1-1697",
+    "Newtonsoft.Json": "9.0.1",
+    "System.Text.Encoding.CodePages": "4.0.1",
+    "System.Diagnostics.FileVersionInfo": "4.0.0",
+    "Microsoft.ApplicationInsights": "2.0.0",
```

Comment:
The reference to dotnet is a PIT thing to bring in the builtin command factory. I'm hoping to move that class so this won't need a `dotnet` dependency at all. Most of these refs will then go away.